java/src/com/google/idea/blaze/java/libraries/DisableLibraryBytecodeNotification.java

```diff
@@ -28,14 +29,14 @@
  * expected for blaze projects, since we're attaching header jars to the project to improve
  * performance.
  */
-final class DisableLibraryBytecodeNotification implements StartupActivity {
+final class DisableLibraryBytecodeNotification extends ProjectActivityJavaShim {
 
   private static final BoolExperiment enabled =
       new BoolExperiment("disable.bytecode.notification", true);
 
   @Override
   public void runActivity(@NotNull Project project) {
-    if (!enabled.getValue() || !Blaze.isBlazeProject(project)) {
+    if (!enabled.getValue() || Blaze.getProjectType(project) == ProjectType.UNKNOWN) {
```

Comment:
why?

Reply:
The method `isBlazeProject` is [deprecated](https://github.com/bazelbuild/intellij/blob/b7bd52f6985014773360fb7fb7bce2f8cefa1bc1/base/src/com/google/idea/blaze/base/settings/Blaze.java#L45) since 2023.